host-controller/src/main/java/org/jboss/as/domain/controller/logging/DomainControllerLogger.java

```diff
@@ -802,4 +802,7 @@
 
     @Message(id = 97, value = "Cannot explode a subdeployment of an unexploded deployment")
     OperationFailedException cannotExplodeSubDeploymentOfUnexplodedDeployment();
+
+    @Message(id = 98, value = "You are trying to redeploy rutimes not affected by the overlay")
```

Comment:
typo

s/rutimes/runtimes

The method should also be redeployingUnaffectedRuntimes (typo in 'affected')